sdk/python/kfp/components/_python_op.py

```diff
@@ -744,7 +745,9 @@ def func_to_container_op(
     extra_code: Optional[str] = '',
     packages_to_install: List[str] = None,
     modules_to_capture: List[str] = None,
-    use_code_pickling: bool = False):
+    use_code_pickling: bool = False,
+    annotations: Mapping[str, str] = None,
```

Comment:
```suggestion
    annotations: Optional[Mapping[str, str]] = None,
```
for consistency, maybe?

Reply:
Done.